server/neptune/workflows/activities/command/subcommand.go

```diff
@@ -17,7 +17,7 @@ const (
 	Show  Operation = "show"
 )
 
-// Argument is the key value pair passed into the terraform command
+// Argument is the key value pair passed into the command
 type Argument struct {
```

Comment:
Is this model still accurate for conftest args? Like do they use `-key=value`

Reply:
Yes when you pass in the path, you can set up the key as `p` and the value as `path`